templates/commands/render_action.go

```diff
@@ -78,3 +79,35 @@ func walkAndModify(pos *model.ConfigPos, rfs renderFS, scratchDir, relPath strin
 		return nil
 	})
 }
+
+func templateAndCompileRegexes(regexes []model.String, inputs map[string]string) ([]*regexp.Regexp, error) {
+	compiled := make([]*regexp.Regexp, len(regexes))
+	for i, re := range regexes {
+		templated, err := parseAndExecuteGoTmpl(re.Pos, re.Val, inputs)
+		if err != nil {
+			return nil, err
+		}
+
+		compiled[i], err = regexp.Compile(templated)
+		if err != nil {
+			return nil, model.ErrWithPos(re.Pos, "failed compiling regex: %w", err) //nolint:wrapcheck
+		}
```

Comment:
Do we want to return a multierror here instead? It could be useful to get _all_ your errors at once instead of "fix error, run again, fix different error, run again, fix different error ..."

```suggestion
		templated, err := parseAndExecuteGoTmpl(re.Pos, re.Val, inputs)
		if err != nil {
		  merr = multierror.Append(merr, err)
			continue
		}

		compiled[i], err = regexp.Compile(templated)
		if err != nil {
			merr = multierror.Append(merr, model.ErrWithPos(re.Pos, "failed compiling regex: %w", err))
			continue
		}
```

Reply:
SGTM, done.